### ISSUE TITLE
make importlib_metadata work across all live python versions

### ISSUE THREAD

**Issue by mwartell** [twosixlabs/armory#1316]
Lucas reports:
```
__init__.py:15: in <module>
    from importlib_metadata import version, PackageNotFoundError
E   ModuleNotFoundError: No module named 'importlib_metadata'
=====================
```
That package is a feature backport and should prefer importlib.metadata for python 3.8+

**Comment by mwartell** [comment]
So this import was introduced with the scm versioning a couple weeks back.  In python3.8+ there is a builtin importlib.metadata package. The `pip install importlib_metadata` is a backport of that feature to 3.7 where it is not in the standard library.

I suspect our use of that line was a paste in from an SCM howto.  I think we should prefer the native version when available thus yielding

    try:
        from importlib.metadata import version ...
    except ModuleNotFoundError:
         from importlib_metadata import version ...

The one thing I don't like about this is I cannot see how `importlib_metadata` got installed since our requirements.txt file doesn't mention it.

**Comment by davidslater** [comment]
`pytest` includes `importlib_metadata` as a dependency in python 3.7 and python 3.6.

If you wanted to, you could add a second `except ModuleNotFoundError` statement that tells the user to pip install requirements.

**Comment by mwartell** [comment]
Fixed by #1327

Since `pytest` unpinned is in `requirements.txt` which is currently an install prerequisite for armory, I'd prefer not adding predicates on which python version you are running.
